include/setjmp.h

```diff
@@ -9,10 +9,21 @@
 
 #ifndef __SETJMP_H
 #define __SETJMP_H
+#include <stdint.h>
+/*
+ * jmp_buf contains the following:
+ * rbx - 1 uintptr_t
+ * r12 to r15 - 4 uintptr_t
+ * rbp - 1 uintptr_t
+ * rsp - 1 uintptr_t
+ * return address - 1 uintptr_t
+ * 2 function arguments - 2 uintptr_t
+ * total = 10 uintptr_t
+ */
 
-typedef int jmp_buf[10];
+typedef uintptr_t jmp_buf[10];
 #ifdef _POSIX_SOURCE
-typedef int sigjmp_buf[10];
+typedef uintptr_t sigjmp_buf[10];
```

Comment:
POSIX compliance. Sigsetjmp requires it, see POSIX_VISIBLE in FreeBSD or in musl:

#if defined(_POSIX_SOURCE) || defined(_POSIX_C_SOURCE) \
 || defined(_XOPEN_SOURCE) || defined(_GNU_SOURCE) \
 || defined(_BSD_SOURCE)
typedef jmp_buf sigjmp_buf;
int sigsetjmp (sigjmp_buf, int);
_Noreturn void siglongjmp (sigjmp_buf, int);

Probably the typedef should be inside the second ifdef _POSIX_SOURCE, leaving only one round of cpp.